apps/web/layouts/auth.vue

```diff
@@ -13,7 +13,7 @@
     </main>
     <NuxtLazyHydrate when-idle>
       <UiNavbarBottom v-if="viewport.isLessThan('md')" />
-      <Cookiebar />
```

Comment:
I am wondering if this will have negative impact on our lighthouse scores, In theory this will always overlap with the content that was already rendered by the ssr server. I run a test and there are no layoutshifts, we still need to check if it impacts the performance. Need to discuss if it would be possible to render it with ssr and not create hydration errors.

Reply:
solved: all cookies are now registered during ssr and the cookiebar is also rendered only on server